### ISSUE TITLE
Bridge has wider shoulder than the road's

### ISSUE THREAD

**Issue by La62ser** [FritzsHero/RoadArchitect#25]
Hi FritzsHero,

I've encountered an issue when creating a bridge. The bridge's shoulder is much wider than the road's. I can't find where or how to make the bridge as wide as the road.

Please see below screenshot:
![road architect - bridge bug](https://user-images.githubusercontent.com/41779443/127442856-e0eb5061-cdae-4bd3-8cbd-275bc161633a.PNG)

Here is my road setting. The Shoulders width is set to 0.5, which is relatively narrow (see above image).
![road setting](https://user-images.githubusercontent.com/41779443/127443122-2ee68216-07c6-456b-8d12-af9face0a586.PNG)

**Version**
Road Architect version: 2.0.2
Unity version: 2021.1.11f1c1

Not sure if it's a bug or not.
Kindly, please help to take a look when you have time.

Thank you very much.

**Comment by RedAlert244** [comment]
Yes, object extrusion code does not imply adapt bridge width to road width. That might be a useful enhancement. But anyway you can adapt it manually. Just modify width of existing bridgebase mesh using any 3d editor and extrude modified mesh in road architect.

**Comment by La62ser** [comment]
@RedAlert244 Got it. Thanks.
